internal/records/report.go

```diff
@@ -37,10 +43,31 @@ type Attrs struct {
 type Upness struct {
 	ReadyCount    int32 `json:"readyCount"`
 	ExpectedCount int32 `json:"expectedCount"`
+	UnknownCount  int32 `json:"unknownCount"`
 	Attrs
 }
 
 func (up Upness) Up() bool {
+	if experiments.IsExperimentEnabled("NodeUnknownAsReady") {
```

Comment:
I would follow with the same convention of passing config down where needed, not transmitting config via global state - this gets messy fast and becomes tedious to test. I would prefer to have an implementation like `Up(unknownThreshold float64)`.